test/tests/button/validation.js

```diff
@@ -124,6 +129,44 @@ let buttonConfigs = [
         }))
     },
 
+    {
```

Comment:
Can we add some tests for the new logos?

Reply:
they are checked in now